packages/venia-concept/src/components/ToastContainer/toastContainer.css

```diff
@@ -0,0 +1,13 @@
+.container {
+    position: fixed;
+    min-width: 100%;
+    display: grid;
+    grid-row-gap: 1rem;
+
+    /* Necessary to display over page contents */
+    z-index: 9999;
```

Comment:
`5` or `6` should be sufficient, I would think. 😂

Reply:
3 years later...

> *[BUG]: Toasts are not displayed over my modal!

Reply:
But I can change it :P